pkg/notes/notes.go

```diff
@@ -68,6 +72,13 @@ type CVEData struct {
 	Description string  `json:"description"`
 }
 
+// MajorTheme struct to define the release major versions
+type MajorTheme struct {
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	SIG         string `json:"sig"`
```

Comment:
I don't think we specifically call out SIGs in Major Theme entries. might be useful to keep track of which SIGs have responded, but if it's too much work to do that then I'm perfectly ok with just deleting SIG field here.